src/Modules/AuthModule/components/Login/Login.tsx

```diff
@@ -1,6 +1,160 @@
+import logo from "../../../../assets/images/PMS 3.svg";
+import './Login.css';
+import { useForm } from "react-hook-form";
+import { NavLink, useNavigate } from "react-router-dom";
+import { FormData } from "../../../../interfaces/Auth.ts";
+import {  useState } from "react";
+import axios from "axios";
+import { toast } from 'react-toastify';
 
+
+// Function 
 export default function Login() {
-  return (
-    <div>Login</div>
-  )
+  const navigate = useNavigate();
+
+  // useForm
+  // FormData : interface (Auth.ts)
+  const {
+    register,
+    handleSubmit,
+    formState: { errors},
+  } = useForm<FormData>();
+
+
+  // Password eye 
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const togglePasswordVisibility = (): void => {
+    setShowPassword((prevState: boolean) => !prevState);
+  };
+
+  // Send Data to Api
+  const onSubmit = async (data: FormData) => {
+
+    try {
+      const response = await axios.post(
+        "https://upskilling-egypt.com:3003/api/v1/Users/Login",
```

Comment:
Having the URL hardcoded  like this is not a good practice, as this means that if we have the URL in 100 component and the URL has changed that we need to manually update all URLS.

An alternative approach to use axios.create to setup an axios instance that have the base URL and any other configuration, and when using this instance, we will pass only the relative path to the base URL.

We can also store the URL's in a constants  file like this one:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/APIs.ts